src/menu_nanox.c

```diff
@@ -48,7 +64,22 @@ UX_FLOW(
         ux_idle_flow,
         &ux_idle_flow_1_step,
         &ux_idle_flow_2_step,
-        &ux_idle_flow_3_step
+        &ux_idle_flow_3_step,
+        &ux_idle_flow_4_step
 );
 
+static void h_expert_toggle()
+{
+	app_mode_set_expert(!app_mode_expert());
+	ux_flow_init(0, ux_idle_flow, &ux_idle_flow_2_step);
+}
+
+void h_expert_update()
```

Comment:
Compilers are inlining functions irrespective of whether they are declared as inline or not. Note that we have to use `__noinline_due_to_stack__` to avoid inlining our functions where it is not appropriate.

And I want this to be as consistent with expert mode in other apps as possible.